packages/thapi/package.py

```diff
@@ -30,6 +30,7 @@ class Thapi(AutotoolsPackage):
     depends_on('libiberty+pic')
     depends_on('libffi')
     depends_on('pkgconfig')
+    depends_on('python', type=('build'))
```

Comment:
```suggestion
    # We add a python dependency at build, because `lttng-gen-tp` need it.
    # We don't add python as a runtime dependency of lttng, to avoid it propagated is a runtime dependency of thapi
/usr/bin/env: 'python': No such file or directory
    depends_on('python', type=('build'))
```